create_derivatives/tests.py

```diff
@@ -77,3 +78,33 @@ def test_run(self, mock_get_object, mock_init):
     def tearDown(self):
         for f in Path(settings.SRC_DIR).iterdir():
             f.unlink()
+
+
+class PDFMakerTestCase(TestCase):
+
+    def setUp(self):
+        tmp_path = Path(settings.TMP_DIR)
+        if not tmp_path.exists():
+            tmp_path.mkdir(parents=True)
+
+    def set_up_bag(self, fixture_directory, bag):
+        """Adds an uncompressed bag fixture to the temp directory and database"""
+        bag_path = join(settings.TMP_DIR, bag)
+        if not Path(bag_path).exists():
+            shutil.copytree(join("create_derivatives", "fixtures", fixture_directory, bag), bag_path)
+            Bag.objects.create(
+                bag_identifier="sdfjldskj",
+                bag_path=bag_path,
+                origin="digitization",
+                as_data="sdjfkldsjf",
+                dimes_identifier=bag,
+                process_status=Bag.JPG2000)
+
+    def test_run(self):
+        bag_id = "3aai9usY3AZzCSFkB3RSQ9"
+        self.set_up_bag("unpacked_bag_with_jp2", bag_id)
+        make_pdf = PDFMaker().run()
+        self.assertTrue(make_pdf)
```

Comment:
I would add some asserts to check if the actual files were created, the return message is correct, Bag data was updated, and whatever else is happening in that `run` method.